Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -17015,6 +17066,29 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
+		2D56C1BA28A367A00081BD25 /* Model */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 2D56C1CE28A367A00081BD25 /* Build configuration list for PBXNativeTarget "Model" */;
+			buildPhases = (
+				2D56C1C128A367A00081BD25 /* Sources */,
+				2D56C1C428A367A00081BD25 /* Frameworks */,
+				2D56C1C728A367A00081BD25 /* Resources */,
+				2D56C1CA28A367A00081BD25 /* Unlock keychain */,
+				2D56C1CB28A367A00081BD25 /* Process entitlements */,
+				2D56C1CC28A367A00081BD25 /* Create Symlink to Alt Root Path */,
+				2D56C1CD28A367A00081BD25 /* Copy XPC Service to Secondary Path */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				2D56C1BB28A367A00081BD25 /* PBXTargetDependency */,
+				2D56C1BD28A367A00081BD25 /* PBXTargetDependency */,
+			);
+			name = Model;
+			productName = Networking;
```

Comment:
This probably doesn't do anything, since you're setting PRODUCT_NAME in ModelService.xcconfig, but it's evidence of your copy-paste. (And it looks like the GPU process is the same way, lol)
```suggestion
			productName = Model;
```

Reply:
Ha! Interesting... there are no UI in Xcode that shows this at all it seems..? Thanks for catching this